PE-8: Update repo BrightAuthor-Plugins/Set-Video-Mode

mxin512 opened the pull request:
- provided the solution of using a custom plugin and plugin message to set video mode
- added README accordingly

Note: the old [setVideoMode.brs](https://github.com/brightsign/BrightAuthor-Plugins/compare/PE-8?expand=1#diff-134d731e7845a07d4c340f3cd4e305a9f44c82c24d617655eb380b8d9b4bf28d) is out of dated and removed, the new implementation is captured in repo https://github.com/brightsign/video-mode-plugin

mxin512 committed:
PE-8: Update repo BrightAuthor-Plugins/Set-Video-Mode

- provided the solution of using a custom plugin and plugin message to set video mode
- added README accordingly

mxin512 committed:
PE-8: Update repo BrightAuthor-Plugins/Set-Video-Mode

- provided the solution of using a custom plugin and plugin message to set video mode
- added README accordingly

mxin512 commented:
@bhastings1019 [b928233](https://github.com/brightsign/BrightAuthor-Plugins/pull/20/commits/b9282330816acf36e0e363a03c8c9173f3368e6f) is updated based on all your comments.

bhastings1019 commented on Set-Video-Mode/README.md:
````diff
@@ -0,0 +1,105 @@
+
+1. Update videomode_plugin.brs with your expected screens settings
+2. Add videomode_plugin.brs to BrightAuthor or BrightAuthor:Connected (Presentation Settings > Support Content > Script Plugin)
+3. Name ```videomode``` as your Script plugin name (which must be the same as the <plugin_name> described in your brs plugin initialization function)
````
This is slightly confusing to read. Could this be reworded with direct references to what should be assigned or named?

mxin512 replied:
This line 11 was added based on docs "... Click the Add Script Plugin button, give the plugin a name (which must be the same as the <plugin_name> described in the initialization function below..." from [BrightAuthor Plugins and Parsers](https://brightsign.atlassian.net/wiki/spaces/DOC/pages/370673619/BrightAuthor+Plugins+and+Parsers). 

Does "3. Assign the name 'videomode' to your Script plugin. Make sure that the assigned name matches the <plugin_name> specified in your BRs plugin initialization function." this sound reasonable to you?

mxin512 replied:
Updated in [162fa7e](https://github.com/brightsign/BrightAuthor-Plugins/pull/20/commits/162fa7ebf3ab7936aa73d8d76479607b8c60f497)

mingjun7 replied:
@mxin512 Just curious are there limitations to see the video tab? I don't see it on my player's DWS.

mxin512 replied:
@mingjun7 I believe that if you are using an audio only player, you won't see the Video Tab. Otherwise, you should see the Video Tab for DWS.

mxin512 committed:
PE-8: Update repo BrightAuthor-Plugins/Set-Video-Mode

- provided the solution of using a custom plugin and plugin message to set video mode
- added README accordingly

mingjun7 reviewed (approved):
LGTM

bhastings1019 commented on Set-Video-Mode/README.md:
```diff
@@ -0,0 +1,105 @@
+
+## Examples of Valid Resolution and Frame Rates
+
+*Note*: this is not a complete list, and your BrightSign player may not support all these modes. Please refer to the documentation and the Video tab of your BrightSign player to see what is supported.
```
cc: @mxin512

mxin512 replied:
Thanks, updated in [b605515](https://github.com/brightsign/BrightAuthor-Plugins/pull/20/commits/b605515a3fd9508edc4210dc5f47d171d60c0024)

mxin512 committed:
PE-8: Update repo BrightAuthor-Plugins/Set-Video-Mode

- provided the solution of using a custom plugin and plugin message to set video mode
- added README accordingly

blshukla commented on Set-Video-Mode/README.md:
```diff
@@ -0,0 +1,105 @@
+# script-plugin-set-video-mode
+
+This repository contains a example of using Script Plugin and Plugin Message feature to set BrightSign VideoMode.
```
** _an_ example

mxin512 replied:
Updated in [730000e](https://github.com/brightsign/BrightAuthor-Plugins/pull/20/commits/730000e99c8289a24d7a7f658c6f0f989cd53661)

blshukla commented on Set-Video-Mode/README.md:
```diff
@@ -0,0 +1,105 @@
+
+1. Update videomode_plugin.brs with your expected screens settings
+2. Add videomode_plugin.brs to BrightAuthor or BrightAuthor:Connected (Presentation Settings > Support Content > Script Plugin)
+3. Add 'videomode' to your Script plugin name. Make sure that the assigned name matches the <plugin_name> specified in your BRs plugin <plugin_name>_Initialize() function. 
```
** Capitalize whole word - _BRS_

mxin512 replied:
Updated in [730000e](https://github.com/brightsign/BrightAuthor-Plugins/pull/20/commits/730000e99c8289a24d7a7f658c6f0f989cd53661)

blshukla reviewed (approved):
Looks good overall.

mxin512 committed:
PE-8: Update repo BrightAuthor-Plugins/Set-Video-Mode

- provided the solution of using a custom plugin and plugin message to set video mode
- added README accordingly
